tests/components/subaru/test_diagnostics.py

```diff
@@ -52,10 +57,15 @@ async def test_device_diagnostics(
 
     diagnostics_fixture = json.loads(load_fixture("subaru/diagnostics_device.json"))
 
-    assert (
-        await get_diagnostics_for_device(hass, hass_client, config_entry, reg_device)
-        == diagnostics_fixture
-    )
+    raw_data = json.loads(load_fixture("subaru/raw_api_data.json"))
```

Comment:
I think the preference is to use snapshots for diagnostics. You can see how I do it in roborock

Reply:
Thanks for the tip! I've updated the diagnostics tests to use snapshots.